lcm/dbg.h

```diff
@@ -5,97 +5,75 @@
 extern "C" {
 #endif
 
+#include <stdarg.h>
 #include <stdio.h>
-#include <string.h>
 #include <stdlib.h>
-#include <stdarg.h>
+#include <string.h>
 
-#define _NORMAL_    "\x1b[0m"
```

Comment:
I'm not sure if there is anything that can be done about it, but this hardly seems like an improvement.

Reply:
Using a auto-formatter does require giving up some things like "block" formatting. That said I can add comments to disable auto-format in some places, if necessary.

Reply:
Actually, clang-format has *several* options for preserving (or even creating, I suspect) this sort of alignment. Alas, it seems that symbol definitions are missing from that category.

I probably *would* disable formatting for at least `DBG_NAMETAB`, given how much worse that got... (I'm actually rather surprised clang-format did that...)